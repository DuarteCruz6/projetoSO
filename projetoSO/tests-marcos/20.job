WRITE [(j7,94)(x9,79)(t6,25)(x5,61)]
WRITE [(u4,94)(w4,34)(t5,7)(u9,34)(w0,93)]
WRITE [(k5,81)(n8,30)(c4,17)(q7,88)(p6,20)]
READ [c4]
READ [x5]
WRITE [(e8,83)(r0,14)(l7,98)]
BACKUP
READ [e8]
WRITE [(v7,90)(r3,26)(w9,0)(p8,1)]
DELETE [t5,t6,w9,q7]
READ [w0]
READ [u4]
DELETE [r0,k5,j7]
WRITE [(y1,96)(f2,9)(w5,59)(p2,51)(a7,7)]
DELETE [n8,p8,p2,w0,x5]
WRITE [(i5,93)(r8,65)(t3,99)(f3,62)]
READ [e8]
WRITE [(w3,69)(c3,70)]
WRITE [(z3,91)(f7,25)(o3,45)(s2,92)(n2,49)]
READ [a7]
WAIT 2000
READ [w4]
SHOW
WRITE [(l3,41)(t4,91)(i8,27)(j1,27)(n3,71)]
WRITE [(o9,28)(e1,90)(m6,65)(b2,100)(t2,77)]
WRITE [(v0,68)(r5,99)(e7,60)]
WAIT 3000
READ [e8]
READ [c3]
DELETE [s2,w3]
READ [l3]
READ [e1]
WRITE [(d0,48)(d9,11)]
WRITE [(q7,62)(t8,86)]
WRITE [(n1,67)]
WRITE [(k7,51)]
WRITE [(s9,76)(h5,69)]
WAIT 1000
DELETE [d9,e8,i5,t2,h5]
WRITE [(i6,98)(p2,0)(s8,97)(h8,27)]
READ [n2]
READ [v0]
DELETE [q7,s9,n3]
DELETE [r5]
WRITE [(x8,86)]
DELETE [u4,a7,u9]
READ [f7]
WRITE [(d2,51)(b1,80)(w2,26)]
WRITE [(x4,24)(z4,16)(j8,84)]
DELETE [z3]
WRITE [(b0,27)(l1,93)(u5,69)(h7,91)(p4,90)]
WRITE [(u7,40)]
WRITE [(s4,71)(q4,32)(m4,15)(o8,45)(q2,23)]
WRITE [(s5,10)(r6,61)]
DELETE [d0,n1,m6,e7,l3]
WRITE [(a4,9)(j2,92)(i7,69)]
SHOW
WRITE [(n9,93)(q6,59)(z2,45)(x7,55)(q1,8)]
WRITE [(y3,39)(x5,18)]
WRITE [(c7,77)]
DELETE [e1,p6,s4]
READ [m4]
WRITE [(h4,31)(p6,51)]
SHOW
DELETE [w5,c3,b1,o8]
WRITE [(u0,63)(g8,90)(z9,93)(w7,23)]
WRITE [(y2,38)(p9,31)]
WRITE [(n4,40)]
WRITE [(b1,77)(f8,9)]
DELETE [w2]
WRITE [(a5,4)(q0,33)(k8,50)(n7,69)]
BACKUP
DELETE [l7,y3,j8,q4,m4]
WRITE [(i2,31)]
READ [y1]
WRITE [(j4,22)(h0,44)(b3,37)(v6,37)(y4,31)]
READ [w7]
DELETE [v0,k7,h4,b3,h8]
WRITE [(v3,18)]
DELETE [t8,o3]
WRITE [(w0,22)(h4,7)(c1,42)(k4,1)]
BACKUP
WRITE [(k6,4)]
DELETE [b2]
WRITE [(r5,11)(d5,39)(a3,48)]
DELETE [w7]
DELETE [c7]
