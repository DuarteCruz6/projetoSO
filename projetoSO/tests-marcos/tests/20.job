WRITE [(e7,44)(l5,62)(m6,62)(f0,3)(o9,55)]
READ [o9]
WRITE [(w2,18)]
READ [l5]
WRITE [(i5,77)(u3,40)]
READ [e7]
WRITE [(e5,78)(d0,17)]
WRITE [(d1,80)(t9,97)(v8,98)(o2,5)(k0,68)]
WRITE [(w6,67)(y8,38)(v5,15)(g6,70)]
READ [i5]
WRITE [(v4,24)(u6,71)(s7,43)(h6,30)]
DELETE [u6,e5,h6]
SHOW
WRITE [(e4,44)(r8,66)(i0,89)(u4,30)(o6,100)]
READ [l5]
SHOW
DELETE [o6,y8,w2,r8,t9]
WRITE [(v3,47)(t6,10)(m3,10)(y5,52)]
WRITE [(r8,1)(f4,15)(j1,37)]
WRITE [(z5,67)(n1,11)(m4,72)]
BACKUP
WRITE [(h8,29)(f6,83)(i7,36)]
WRITE [(t9,84)]
READ [s7]
READ [t9]
WRITE [(z7,50)(b6,55)]
WAIT 2000
WRITE [(w8,75)(f9,85)(q3,100)(n9,48)]
DELETE [q3]
READ [s7]
BACKUP
WAIT 2000
BACKUP
DELETE [s7,d0]
WRITE [(j3,56)]
WRITE [(r6,42)(j6,69)(l3,12)(m7,76)]
WRITE [(u9,59)(i3,2)(k5,99)]
BACKUP
DELETE [u9,b6]
WRITE [(q0,98)]
WAIT 2000
WRITE [(u5,9)(a7,69)]
WAIT 2000
READ [w8]
WRITE [(w3,43)]
DELETE [e7,m3,m7,i0,y5]
DELETE [a7]
READ [k5]
WRITE [(r5,3)(x0,64)(d8,61)(j7,67)(e2,21)]
DELETE [i5,t9,k5]
READ [h8]
READ [v3]
WRITE [(y6,23)(n7,60)(h0,98)(a0,23)(g4,11)]
WRITE [(l7,46)(i1,33)(y4,6)(w5,54)]
WRITE [(t9,30)]
WRITE [(q4,52)(p2,71)(o4,94)]
