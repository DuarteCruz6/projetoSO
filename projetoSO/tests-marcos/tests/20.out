[(o9,55)]
[(l5,62)]
[(e7,44)]
[(i5,77)]
(d1, 80)
(d0, 17)
(e7, 44)
(f0, 3)
(g6, 70)
(i5, 77)
(k0, 68)
(l5, 62)
(m6, 62)
(o2, 5)
(o9, 55)
(s7, 43)
(t9, 97)
(u3, 40)
(v4, 24)
(v5, 15)
(v8, 98)
(w6, 67)
(w2, 18)
(y8, 38)
[(l5,62)]
(d1, 80)
(d0, 17)
(e4, 44)
(e7, 44)
(f0, 3)
(g6, 70)
(i0, 89)
(i5, 77)
(k0, 68)
(l5, 62)
(m6, 62)
(o6, 100)
(o2, 5)
(o9, 55)
(r8, 66)
(s7, 43)
(t9, 97)
(u4, 30)
(u3, 40)
(v4, 24)
(v5, 15)
(v8, 98)
(w6, 67)
(w2, 18)
(y8, 38)
[(s7,43)]
[(t9,84)]
[(s7,43)]
[(w8,75)]
[(k5,99)]
[(h8,29)]
[(v3,47)]
